app/views/users/_followers.html.haml

```diff
@@ -1,6 +1,10 @@
 %article
   -if @user.followers.empty?
     %p No one is following #{@user.username} yet.
-  - @user.followers.each do |follower|
-    %p= avatar(follower.email)
-    = follower.username
\ No newline at end of file
+  -@user.followers.each do |follower|
+    - if follower.username == current_user.username
```

Comment:
Better move that if condition one line down... You'd be printing the avatar of `follower.email` anyways